### PR TITLE
Add Collabshot.app v1.2.3.985

### PR DESCRIPTION
**Pull request by cmw**
Collabshot is a collaborative screenshot app, developed by Toptal.
The download link is unversioned and will always provide the latest version,
so there is no check for the sha256.

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/collabshot.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'collabshot' do
+  version :latest
```
Comment (vitorgalvao) [inline Casks/collabshot.rb]: `1.2.3.985`

File: Casks/collabshot.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  sha256 :no_check
```
Comment (vitorgalvao) [inline Casks/collabshot.rb]: Add shasum.

File: Casks/collabshot.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+  url 'https://api.collabshot.com/desktop/latest/mac'
```
Comment (vitorgalvao) [inline Casks/collabshot.rb]: `https://collabshot.s3.amazonaws.com/desktop/builds/mac/Collabshot_#{version}.dmg`

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Comment by vitorgalvao** [comment]
Merged. Thank you.

**Comment by cmw** [comment]
Thank you!

**Pull request merged**